examples/meta/src/evaluation/accuracy_multiclass.sg.in

```diff
@@ -0,0 +1,25 @@
+File f_feats_train = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
```

Comment:
the filename is inconsistent with the one below. Look at existing ones and try to do the same style

Reply:
and if it is not consistent, try to improve the naming :)

Reply:
Does this work?

Reply:
appologies for the many commits, I had alot of conflicts with this PR and the other one :(